lib/udp/udp.ml

```diff
@@ -36,39 +42,44 @@ module Make(Ip: V1_LWT.IP) = struct
 
   let id {ip} = ip
```

Comment:
I don't think this is used any longer. It's probably worth adding `true: warn(A-4)` to the `_tags` file to catch unused code (and other problems).